src/chrome/content/rules/DepFile.com.xml

```diff
@@ -1,37 +1,41 @@
 <!--
 	Problematic subdomains:
 
-	al (404)
-	bl (404)
-	cl (Invalid Certificate)
-	dl (Invalid Certificate)
-	el (404)
-	fl (404)
-	gl (404)
-	hl (404)
-	il (Invalid Certificate)
-	jl (404)
-	mn (Apache 2 Test Page, CentOS)
-	nn (Apache Directory Listing, Ubuntu)
-	ol (404)
-	openbtc (Apache 2 Test Page, CentOS + Self-Signed Certificate)
-	pl (404)
-	rn (Non-existent)
-	secure (Travis Fetch error: (60, 'SSL certificate problem: unable to get local issuer certificate') )
-	secured (403)
-	tl (404)
-	ul (cPanel Error Page + Self-Signed Certificate)
-	vl (404)
-	wl (Invalid Certificate)
-	wn (No certificate)
-	www (Certificate only valid for depfile.com.  Redirects.)
-	www.secure (Non-existent)
-	xl (Non-existent)
-	yl (404)
+	Invalid certificate:
+		cl (Invalid Certificate)
+		dl (Invalid Certificate)
+		il (Invalid Certificate)
+		secure (Travis Fetch error: (60, 'SSL certificate problem: unable to get local issuer certificate') )
+		mail.depfile.com (Self-Signed)
+		qn.depfile.com (Self-Signed)
+		ul (cPanel Error Page + Self-Signed)
+		wl (Invalid Certificate)
+		www.depfile.com (Certificate only valid for depfile.com.  Redirects.)
+		www.depfile.us (Invalid certificate)
+		www.dipfile.com (Invalid certificate)
+
+	404:
+		al (404)
+		el (404)
+		ol (404)
+		tl (404)
+		vl (404)
+		yl (404)
+
+	Other:
+		mn (Apache 2 Test Page, CentOS)
+		nn (Apache Directory Listing, Ubuntu)
+		openbtc (Apache 2 Test Page, CentOS + Self-Signed Certificate)
+		www.secure (Non-existent)
```

Comment:
You can safley remove it, since no DNS record exits for this host.
but there is still one for `secured.depfile.com`